pkg/types/target.go

```diff
@@ -14,6 +14,12 @@ type Scanner string
 // Scanners is a slice of scanners
 type Scanners []Scanner
 
+// Runtime represents a container runtime
+type Runtime string
+
+// Runtimes is a slice of runtimes
+type Runtimes []Runtime
```

Comment:
It should be defined under ImageOptions.
https://github.com/aquasecurity/trivy/blob/9bc326909fce6f5a591eaded17a26a0560d4ad8d/pkg/fanal/types/image.go#L17-L22

Reply:
done